usaspending_api/tests/conftest_spark.py

```diff
@@ -204,7 +204,7 @@ def hive_unittest_metastore_db(spark: SparkSession):
     spark.sql("USE DEFAULT")
     tables_in_default_schema = [t for t in spark.sql("SHOW TABLES").collect()]
     for t in tables_in_default_schema:
-        spark.sql(f"DROP TABLE IF EXISTS {t['tableName']} CASCADE")
```

Comment:
Why remove the CASCADE? Should be able to keep this place.

Reply:
That was a test i did that shouldn't have been pushed, replaced